server/index.js

```diff
@@ -11,6 +11,8 @@ import Board from '../client/js/board.js';
 
 var b = new Board();
 
+const snakes = [];
```

Comment:
The board already contains an array of snakes https://github.com/ecvdbdx1617/MultiSnake/blob/97100e290d41fb761e0897dd3183e08a64dd1fb1/client/js/board.js#L13
How is this array different than the board one?